src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -1,21 +1,46 @@
-﻿<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <NoWarn>$(NoWarn);44;62;69;65;54;61;75;62;9;2003;NU5125</NoWarn>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
+    <OutputType>Library</OutputType>
+    <NoWarn>$(NoWarn);44;45;54;55;61;62;69;65;75;1204;2003;NU5125</NoWarn>
+    <AssemblyName>FSharp.Compiler.Service</AssemblyName>
```

Comment:
the default is the project name so this is redundant

Reply:
May I ask why net472 was added here?